README.md

```diff
@@ -56,6 +56,8 @@ await app.register(fastifyOtelInstrumentation.plugin());
 app.get('/', () => 'hello world')
 // as well as your instance level hooks.
 app.addHook('onError', () => /* do something */)
+// Manually skip telemetry for a specific route
+app.get('/healthcheck', { config: { otel: false } }, () => 'Up!')
```

Comment:
Feel free to keep it as `nit`; I'd like to suggest an alternative, maybe instead of `otel` as an scalar value, we can create an object structure so can be extended in the future if required.

e.g. `otel: { instrument: false, other: 'option' }`

Reply:
`false | { other: 'option' }` could also be implemented in the future if needed since this is JavaScript 😭😅